ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -235,6 +235,12 @@ public override void SetDimensions(Border view, Dimensions dimensions)
 
         private void OnImageFailed(Border view)
         {
+            if (!view.HasTag())
```

Comment:
Unrelated change? If so, can you get a separate PR (if we ever have to revert the Control stuff, might be good to keep this alive).

Reply:
Well, more or less. I added a better cleanup of the tag/etc. data associated with views (we were kind of leaking them in some scenarios), and as an effect these async completions may find unmounted views.
It's all about when we call view.ClearData(); I had added one to NativeViewHierarchyManager.DropView().  

---
In reply to: [159515972](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159515972) [](ancestors = 159515972)

Reply:
No worries, leave this here then.

---
In reply to: [159517763](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159517763) [](ancestors = 159517763,159515972)

Reply:
Could the tag cleanup go entirely in a separate review?

---
In reply to: [159519014](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159519014) [](ancestors = 159519014,159517763,159515972)

Reply:
Ok, will do, makes sense. The ClearData+Image change set are orthogonal to the current matter.

---
In reply to: [159519074](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159519074) [](ancestors = 159519074,159519014,159517763,159515972)